src/bic/router.js

```diff
@@ -47,7 +47,7 @@ define(function (require) {
       this.middleware.addErrorHandler(Middleware.errorHandler);
 
       this.isOfflineFirst = (function () {
-        var isLocalProtocol = location.protocol === 'file:' || location.protocol === 'ms-appx:';
+        var isLocalProtocol = ['file:', 'ms-appx:', 'ms-appx-web'].indexOf(location.protocol) > -1;
```

Comment:
The original 2 protocols have a colon on the end, but the new one doesn't. Is that intentional?